SchnorrMusigSDKSwift/Classes/SchnorrMusig.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  SchnorrMusig.swift
+//  SchnorrMusigSDKSwift
+//
+//  Created by Eugene Belyakov on 04/04/2021.
+//
+
+import Foundation
+
+
+public class SchnorrMusig {
+
+    public init() {}
+    
+    public func createSigner(publicKeys: [Data], position: Int) -> SchnorrMusigSigner {
+        let encodedKeys = publicKeys.reduce(into: Data()) { $0.append($1) }
+        return createSigner(encodedPublicKeys: encodedKeys, position: position)
+    }
+    
+    public func createSigner(encodedPublicKeys: Data, position: Int) -> SchnorrMusigSigner {
+        encodedPublicKeys.withUnsafeBytes { (encodedPublicKeysRaw) in
+            let encodedPublicKeysBuffer = encodedPublicKeysRaw.baseAddress!.assumingMemoryBound(to: UInt8.self)
+            let musig = schnorr_musig_new_signer(encodedPublicKeysBuffer, encodedPublicKeys.count, position)
+            return SchnorrMusigSigner(signer: musig!)
+        }
+    }
+
+    public func verify(message: Data, signature: SMAggregatedSignature, aggregatedPublicKeys: SMAggregatedPublicKey) throws -> Bool {
+        return try self.verify(message: message, signature: signature, encodedPublicKeys: aggregatedPublicKeys.data())
```

Comment:
Please rename `aggregatedPublicKeys` plural to singular (without `s`), as users should pass only 1 agg. pubkey into this function

Reply:
Done